### PR TITLE
Retr0h/improved linting

### PR DESCRIPTION
**Pull request by retr0h**

### CODE REVIEW

File: molecule/validators.py [hunk 1]
```diff
@@ -35,14 +35,19 @@ def check_trailing_cruft(ignore_paths=[]):
     filenames = []
     pruned_filenames = []
     found_error = False
+    valid_extensions = ['py', 'yml', 'rb']
```
Comment (abrown-sg) [inline molecule/validators.py]: Should this be a config option?

**Comment by abrown-sg** [comment]
Looks good. This probably addresses, more or less, CIA-133.

File: molecule/validators.py (hunk 1 above)
Comment (retr0h) [inline molecule/validators.py]: Thought about it, but not really sure it mattes.  Was mostly trying to keep the config from growing excessively, and couldn't think of a good reason people would be changing this.  I assume we can make it a config option if it displeases people ;)  This was me asserting some conventions w/o the configuration :)

**Comment by rgreinho** [comment]
Just my 2 cents here, but a comprehension would be nicer here I think:
```
filenames.append([os.path.join(root, name) for name in files if name.split(os.extsep)[-1] in valid_extensions])
```

This is just a matter of opinion, so I give :+1:  anyway

**Comment by retr0h** [comment]
> Just my 2 cents here, but a comprehension would be nicer here I think:

I agree.  I try to use list comprehension wherever possible.  In this case I thought it was doing a bit much, but after looking at it more, I like it. Updated PR.

**Comment by rgreinho** [comment]
:+1:

**Pull request merged**